gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -285,7 +285,9 @@ end
 -- @return boolean
 -- @realm shared
 function plymeta:IsInTeam(ply)
-	return self:GetTeam() ~= TEAM_NONE and not TEAMS[self:GetTeam()].alone and self:GetTeam() == ply:GetTeam()
+	return self:GetTeam() ~= TEAM_NONE
+	and not TEAMS[self:GetTeam()].alone
```

Comment:
I'm not sure if this is more readable than it was previously. Maybe add an additional tab to this and the next line?